### PR TITLE
Feature/watch message

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
[Request Twitter Authentication through Local Storage [2]](https://trello.com/c/lCsJa7lN/4529-request-twitter-authentication-through-local-storage-2)

File: src/watch.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+function receiveContentFile(message) {
+  if (["DELETED", "NOEXIST"].includes(message.status)) {return;}
+
+  return platform.readTextFile(message.ospath)
+  .then(fileData=>{
+    try {
+      const fileJSON = JSON.parse(fileData);
+
+      logger.file(`Setting companyId ${JSON.stringify(fileJSON.content.schedule.companyIdd)}`);
+      config.setCompanyId(fileJSON.content.schedule.companyId);
```
Comment (jungeunyoon) [inline src/watch.js]: @rodrigopavezi Also think we need some extra validation, thanks. The catch block will catch most errors but making sure that CompanyId isn't null, etc.

File: src/watch.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+function receiveContentFile(message) {
+  if (["DELETED", "NOEXIST"].includes(message.status)) {return;}
+
+  return platform.readTextFile(message.ospath)
+  .then(fileData=>{
+    try {
+      const fileJSON = JSON.parse(fileData);
+
+      logger.file(`Setting companyId ${JSON.stringify(fileJSON.content.schedule.companyIdd)}`);
```
Comment (jungeunyoon) [inline src/watch.js]: @rodrigopavezi Typo here --> ` companyIdd`

**Comment by rodrigopavezi** [comment]
@jungeunyoon I have addressed your comments. Could you please review? Cheers

**Comment by jungeunyoon** [comment]
@rodrigopavezi looks good, thanks 👍

**Pull request merged**